src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomSerializerBase.TraceIfInterpolatedStringHandler.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace System.ComponentModel.Design.Serialization
+{
+    public abstract partial class CodeDomSerializerBase
+    {
+        /// <summary>
+        ///  Provides an interpolated string handler for <see
+        ///  cref="CodeDomSerializerBase.TraceIf(System.Diagnostics.TraceLevel,bool,ref System.ComponentModel.Design.Serialization.CodeDomSerializerBase.TraceIfInterpolatedStringHandler)"
+        ///  /> that only performs formatting if the condition applies and tracing is set to verbose.</summary>
```

Comment:
Should it be something like this?
```suggestion
        ///  /> that only performs formatting if the condition applies and tracing is set to at least specified level.</summary>
```